pristine-zen-garden.html

```diff
@@ -21,83 +21,75 @@
       </header>
       <div class="summary" id="zen-summary" role="article">
         <p>
-        A demonstration of what can be accomplished through <abbr title="Cascading Style Sheets"> CSS</abbr>-based design. Select any style sheet from the list to load it into this page.
+          A demonstration of what can be accomplished through <abbr title="Cascading Style Sheets"> CSS</abbr>-based design. Select any style sheet from the list to load it into this page.
         </p>
         <p>
-        Download the example <a href="/examples/index" title="This page's source HTML code, not   to be modified.">html file</a> and <a href="/examples/style.css" title="This page's sample CSS, the file you may modify.">css file</a>
+          Download the example <a href="/examples/index" title="This page's source HTML code, not   to be modified.">html file</a> and <a href="/examples/style.css" title="This page's sample CSS, the file you may modify.">css file</a>
         </p>
       </div>
 <!-- Preamble-->
       <div class="preamble" id="zen-preamble" role="article">
-        <h3>
-          The Road to Enlightenment
-        </h3>
-        <p>
-        Littering a dark and dreary road lay the past relics of browser-specific tags, incompatible <abbr title="Document Object Model">DOM</abbr>s, broken <abbr title="Cascading Style Sheets">CSS</abbr> support, and abandoned browsers.
-        </p>
-        <p>
-        We must clear the mind of the past. Web enlightenment has been achieved thanks to the tireless efforts of folk like the <abbr title="World Wide Web Consortium">W3C</abbr>, <abbr title="Web Standards Project">WaSP</abbr>, and the major browser creators.
-        </p>
-        <p>
-          The CSS Zen Garden invites you to relax and meditate on the important lessons of the masters. Begin to see with clarity. Learn to use the time-honored techniques in new and invigorating fashion. Become one with the web.
-        </p>
+        <h3>The Road to Enlightenment</h3>
+          <p>
```

Comment:
h3 and p elements have the same div parent, hence should have the same indentation